gtfs/__main__.py

```diff
@@ -1,25 +1,46 @@
 #!/usr/bin/env python
 """Command line interface for fetching GTFS."""
-import argparse
-import getpass
+import typer
+from typing_extensions import Annotated
+from chalky import chain
 import logging
-import sys
-
-from prettytable import PrettyTable
-
-from gtfs.utils.FeedSource import FeedSource
-import feed_sources
-# import all the available feed sources
-# pylint: disable=I0011,wildcard-import
-from feed_sources import *
+from prettytable.colortable import ColorTable, Themes
+from halo import Halo
+from .feed_source import FeedSource
+from .feed_sources import __all__ as feed_sources
 
 logging.basicConfig()
 LOG = logging.getLogger()
 LOG.setLevel(logging.INFO)
+app = typer.Typer()
+            
+@app.command()
+def list_feeds(bbox: Annotated[str, typer.Argument(help="pass value as a string separated by commas like this: xmin,ymin,xmax,ymax ")],
+    predicate: Annotated[str, typer.Option(help="intersects or contains")] = 'intersects'):
```

Comment:
does `predicate` only allow `intersects` or `contains`? if not, we'll need to restrict that, I'm sure there's an arg to `typer.Option` for that?